src/actionCreator.ts

```diff
@@ -57,3 +58,11 @@ export function getPrivateActionId(target: any) {
 function setPrivateActionId(target: any, actionId: string) {
     target.__SATCHELJS_ACTION_ID = actionId;
 }
+
+export function getActionType(target: any): string {
```

Comment:
It's a nitpick, but can you follow the same pattern as we do for action ID:
1. Name the property `__SATCHELJS_ACTION_TYPE`
2. Name the getter/setter `getPrivateActionType` and `setPrivateActionType`.

Other than that, this looks great!  Thanks for fixing it.

Reply:
Done!